_config.yml

```diff
@@ -29,6 +29,9 @@ navigation:
 - text: Collaborating on Code
```

Comment:
Agreed. I don't quite know where this repository came from... Maybe "Collaborating on Code" should become "Main"/"General"/"Overview"?